frontend/src/APIUtils.js

```diff
@@ -0,0 +1,57 @@
+import { SERVER_ADDRESS, SERVER_PORT, ROOM_ROUTES } from './shared_code/consts';
+import axios from 'axios';
+
+export const createRoomFromAPI = async (roomDispatch, creatorID) => {
```

Comment:
no need to write FromApi, it doesn't mean anything

Reply:
https://github.com/aviadch/Mafia/pull/9/commits/85bf741eae06a303f462de258fbe8438ff46e26b